FirebaseAuth/Sources/Public/FirebaseAuth/FIRUser.h

```diff
@@ -160,7 +162,8 @@ NS_SWIFT_NAME(User)
  */
 - (void)updateEmail:(NSString *)email
          completion:(nullable void (^)(NSError *_Nullable error))completion
-    NS_SWIFT_NAME(updateEmail(to:completion:));
+    NS_SWIFT_NAME(updateEmail(to:completion:))
+        DEPRECATED_MSG_ATTRIBUTE("Use verifyBeforeUpdateEmail() instead.");
```

Comment:
I'm not seeing an existing API named `verifyBeforeUpdateEmail()`. Did you mean `sendEmailVerificationBeforeUpdatingEmail:completion:`?

Reply:
Yes, thanks for catching this.